saleor/account/models.py

```diff
@@ -200,6 +200,11 @@ class Meta:
                 fields=["search_document"],
                 opclasses=["gin_trgm_ops"],
             ),
+            GinIndex(
+                name="user_p_meta_jsonb_path_idx",
```

Comment:
You can make the index name clearer, max length is 63 chars:
```suggestion
                name="user_private_metadata_jsonb_path_idx",
```

Reply:
@NyanKiyoshi we use `_p_meta_` as a short in a lot of other places (apps, users, gift cards, checkouts, invoices, menus, and so on) so I kept it like that for consistency.